test/sqllogictest/types.slt

```diff
@@ -674,13 +674,113 @@ time
 query T
 SELECT format_type(1114, NULL)
 ----
-timestamp
+timestamp without time zone
+
+query T
+SELECT format_type(1114, -2)
+----
+timestamp without time zone
+
+query T
+SELECT format_type(1114, -1)
+----
+timestamp without time zone
+
+query T
+SELECT format_type(1114, 0)
+----
+timestamp(0) without time zone
+
+query T
+SELECT format_type(1114, 1)
+----
+timestamp(1) without time zone
+
+query T
+SELECT format_type(1114, 2)
+----
+timestamp(2) without time zone
+
+query T
+SELECT format_type(1114, 3)
+----
+timestamp(3) without time zone
+
+query T
+SELECT format_type(1114, 4)
+----
+timestamp(4) without time zone
+
+query T
+SELECT format_type(1114, 5)
+----
+timestamp(5) without time zone
+
+query T
+SELECT format_type(1114, 6)
+----
+timestamp(6) without time zone
+
+query T
+SELECT format_type(1114, 7)
+----
+timestamp(7) without time zone
```

Comment:
@sploiselle Does this look sane? 7 is an invalid value here.

Reply:
Ideally, we just match Postgres, which this does so is 👌 

```
postgres=# SELECT format_type(1114, 7);
          format_type           
--------------------------------
 timestamp(7) without time zone
```

Reply:
Cool 👍